var/www/html/consul-templates/db-config.php.ctmpl

```diff
@@ -0,0 +1,457 @@
+<?php
```

Comment:
I have mixed feelings about putting these in a directory called "consul-templates". In a lot of ways, `consul-template` is an implementation detail that is definitely lower-priority than the rest of the application. And, keeping the templates in the same dir that people look for the real config file might help them figure out how the app works.